File: 0205 isomorphic-strings/solution.cs
public class Solution {
    public bool IsIsomorphic(string s, string t) {
        if(s.Length!=t.Length) return false;
        int[] a=new int[300];
        int[] b=new int[300];
        for(int i=0;i<s.Length;i++){
            if(a[s[i]]!=b[t[i]]) return false;
            a[s[i]]=i+1;
            b[t[i]]=i+1;
        }
        return true;
    }
}

File: 0205 isomorphic-strings/solution.cpp
class Solution {
public:
    bool isIsomorphic(string s, string t) {
        if(s.size() != t.size()) return false;
        vector<int> a(256, -1); 
        vector<int> b(256, -1); 
        for(int i=0;i<s.size();i++) {
            if(a[s[i]]!=b[t[i]]) return false;
            a[s[i]]=i;
            b[t[i]]=i;
        }
        return true;
    }
};
